samples/AppConfiguration/readme.md

```diff
@@ -6,6 +6,6 @@
 title: AppConfiguration
 generation1-convenience-client: true
 require: $(this-folder)/../../readme.md
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/e606243e5297312781dd7dbfd7ab76d2329cc088/specification/appconfiguration/data-plane/Microsoft.AppConfiguration/stable/1.0/appconfiguration.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/a63ddbed578f026d4e140345d240eff90cdd67ce/specification/appconfiguration/data-plane/Microsoft.AppConfiguration/stable/1.0/appconfiguration.json
```

Comment:
just curious, why do we update the commit id of the sample spec? Is that because we want to sync up with the latest spec?

Reply:
Yep. Just to be sure this PR doesn't break anything.

Reply:
But you're right, it is not required for this PR. Reverting.